critiquebrainz/db/users.py

```diff
@@ -0,0 +1,17 @@
+from critiquebrainz import db
+from critiquebrainz.db import exceptions as db_exceptions
+import sqlalchemy
+
+
+def getusers_id(usernames):
+    """Get MusicBrainz ids of users
+    Args:
+        usernames (list): MusicBrainz usernames.
+    """
+    with db.engine.connect() as connection:
+        req = sqlalchemy.text("""SELECT * FROM "user" WHERE display_name IN :users""")
+        result = connection.execute(req, users=tuple(usernames))
```

Comment:
Also make sure to explicitly define which columns are returned there instead of just `SELECT *`. Refer to the example I linked.